src/components/ChataTable/TableWrapper.js

```diff
@@ -26,7 +26,7 @@ export default class TableWrapper extends React.Component {
       reactiveData: false,
       autoResize: true,
       rowHeight: 25,
-      layout: this.props.pivot ? 'fitDataFill' : 'fitColumns',
+      layout: this.props.isDrilldown ? 'fitDataFill' : (this.props.scope === 'dashboards' ? 'fitColumns' : 'fitDataFill'),
```

Comment:
What was the issue with using fitColumns for drilldowns

Reply:
for drilldowns the table has lots of columns to view the data and it's better to use the fitDataFill, otherwise some numeric column's width will be minimal width automatically and affect the view.
Our initial purpose of this feature is to help the resize window scenario. 
Drilldown and other UI such as notification, data messenger, which are using tables are actually not being resized.

Reply:
<img width="1454" height="480" alt="Screenshot 2025-07-22 at 3 13 59 PM" src="https://github.com/user-attachments/assets/32775513-79e9-4a62-b2e7-9ad87ddd7011" />
but if we use `fitDataFill `, the above image demos that it has an expected UI.

Reply:
Okay, thanks for the explanation Alex. Sounds good